server/routes/juror-management/juror-record/juror-record.controller.js

```diff
@@ -1036,6 +1037,41 @@
     };
   };
 
+  module.exports.getHistoryTab = (app) => async(req, res) => {
+    try {
+      const historyTab = Object.keys(req.query)[0] || 'history';
+      const history = (historyTab === 'history' 
+        ? (await jurorHistoryDAO.get(req, req.params.jurorNumber)).data.sort((a,b) => b.dateCreated.localeCompare(a.dateCreated))
+        : (await jurorPaymentsHistoryDAO.get(req, req.params.jurorNumber)));
+
+      const jurorNumber = req.params.jurorNumber;
+      const {data: juror} = await jurorRecordObject.record.get(
+        require('request-promise'),
+        app,
+        req.session.authToken,
+        'overview',
+        jurorNumber,
+        req.session.locCode,
+      );
+
```

Comment:
we should probably have some logging on these calls if they succeed

Reply:
Added